src/node.cc

```diff
@@ -3403,6 +3409,14 @@ static void ParseArgs(int* argc,
       printf("%s\n", NODE_VERSION);
       exit(0);
     } else if (strcmp(arg, "--help") == 0 || strcmp(arg, "-h") == 0) {
+      if (strcmp(NODE_PLATFORM, "win") != 0 &&
+          strcmp(NODE_PLATFORM, "win32") != 0) {
```

Comment:
Is this check perhaps a bad idea if people are running something cygwin or git bash on windows?